unicodetools/src/main/java/org/unicode/xml/UCDPropertyDetail.java

```diff
@@ -1884,9 +1884,9 @@ public class UCDPropertyDetail {
                     true,
                     false,
                     true);
-    public static UCDPropertyDetail kRSTUnicode_Detail =
+    public static UCDPropertyDetail kTGT_RSUnicode_Detail =
```

Comment:
@markusicu - I think that the best way of handling this is to add a line for maxVersion as VersionInfo.getInstance(16, 0, 0), then have a new UCDPropertyDetail for kTGT_RSUnicode_Detail (and others) with the minVersion as VersionInfo.getInstance(17, 0, 0). In that way, we'll still be able to build older versions of UCDXML.

Reply:
Having said that, if you want to complete this PR as is, I'll go back and implement this change after the fact.

Reply:
Not sure what you mean. This code uses named objects from UcdProperty which are changing names. It cannot compile with `UcdProperty.kRSTUnicode` because that object name no longer exists.

Reply:
Ugh. That will mean that we won't be able to build 16.0.0 or previous UCDXML files without some hackery...